lib/console.js

```diff
@@ -17,4 +17,7 @@ module.exports = function (context, _instances) {
   });
   tabl.push(['TOTALS', output.totals.pass, output.totals.fail, output.totals.total]);
   console.log(tabl.toString());
+  if (context.program.exit) {
+    process.exit(Math.min(output.totals.fail, 255));
```

Comment:
Since you're modifying this code, do you want to consolidate the changes in from #30 ?

Reply:
Sure Matt will do